tests/fullydistributed_grids/memory_consumption_01.cc

```diff
@@ -0,0 +1,82 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 - 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+// Monitor memory consumption.
+
+#include <deal.II/base/mpi.h>
+
+#include <deal.II/distributed/fully_distributed_tria.h>
+#include <deal.II/distributed/shared_tria.h>
+#include <deal.II/distributed/tria.h>
+
+#include <deal.II/fe/fe_q.h>
+
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/grid_out.h>
+#include <deal.II/grid/grid_tools.h>
+#include <deal.II/grid/tria.h>
+
+#include "./tests.h"
+
+using namespace dealii;
+
+template <int dim>
+void
+test(MPI_Comm comm)
+{
+  // create serial triangulation
+  Triangulation<dim> basetria;
+
+  GridGenerator::subdivided_hyper_cube(basetria, 10);
+  basetria.refine_global(2);
+
+  GridTools::partition_triangulation_zorder(
+    Utilities::MPI::n_mpi_processes(comm), basetria);
+
+  // create instance of pft
+  parallel::fullydistributed::Triangulation<dim> tria_pft(comm);
+
+  // register dynamic construction data
+  auto construction_data =
+    TriangulationDescription::Utilities::create_description_from_triangulation(
+      basetria, comm);
+
+  // actually create triangulation
+  tria_pft.create_triangulation(construction_data);
+
+  auto min_max_avg =
+    Utilities::MPI::min_max_avg(tria_pft.memory_consumption(), comm);
+
+  Assert((min_max_avg.min / min_max_avg.max >= 0.75),
+         ExcMessage("Memory consumption difference is too big!"));
+
+  deallog << "OK!" << std::endl;
+}
+
+int
+main(int argc, char *argv[])
+{
+  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
+  MPILogInitAll                    all;
```

Comment:
Since we do not need to the specific output any more, how about:
```suggestion
  mpi_initlog();
```

Reply:
Done!